src/js/common.js

```diff
@@ -86,6 +86,10 @@ module.exports.getPathCommand = (searchableName) => lookup[searchableName].pathC
 module.exports.detectOS = () => {
   return platforms.find((aPlatform) => {
     /*global platform*/
+    // Workaround for Firefox on macOS which is 32 bit only
```

Comment:
Firefox on my Mac is 64 bit....

Reply:
some versions of firefox report both 32 and 64 in the string meaning that the website tries to serve a 32 bit download.

Reply:
OK

Reply:
Ok